File: lat3_PHP/Disk.php
<?php

class Disk{
    private $Type = "";
    private $Capacity = "";
    private $Price = 0;
    
    public function __construct($Type = "", $Capacity="", $Price = 0){
        $this->Type = $Type;
        $this->Capacity = $Capacity;
        $this->Price = $Price;
    }

    public function setType($Type){
        $this->Type = $Type;
    }

    public function getType(){
        return $this->Type;
    }

    public function setCapacity($Capacity){
        $this->Capacity = $Capacity;
    }

    public function getCapacity(){
        return $this->Capacity;
    }

    public function setPrice($Price){
        $this->Price = $Price;
    }

    public function getPrice(){
        return $this->Price;
    }

    public function printDisk(){
        echo "Disk Type         : ". $this->getType() ."<br>";
        echo "Disk Capacity     : ". $this->getCapacity() ."<br>";
        echo "Disk Price        : ". $this->getPrice() ."<br>";
    }

}  
?>

File: lat3_java/disk.java
class disk{
    private String type;
    private String capacity;
    private int price;
    
    public disk() {
    }

    public disk(String type, String capacity, int price) {
        this.type = type;
        this.capacity = capacity;
        this.price = price;
    }

    public int getPrice() {
        return price;
    }

    public void setPrice(int price) {
        this.price = price;
    }

    public String getCapacity() {
        return capacity;
    }

    public void setCapacity(String capacity) {
        this.capacity = capacity;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public void printDisk(){
        System.out.print("Disk Type       : ");
        System.out.println(getType());
        System.out.print("Disk Capacity   : ");
        System.out.println(getCapacity());
        System.out.print("Disk Price      : ");
        System.out.println(getPrice());
    }

}